src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -0,0 +1,238 @@
+      val dateCreated = dateMap("created")
+      val dateAvailable = {
+        val elems = dateMap("available")
+        if (elems.isEmpty) dateCreated
+        else elems
+      }
+      <ddm:DDM
+        xmlns:dc="http://purl.org/dc/elements/1.1/"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:dct="http://purl.org/dc/terms/"
+        xmlns:dcx-dai="http://easy.dans.knaw.nl/schemas/dcx/dai/"
+        xmlns:dcx-gml="http://easy.dans.knaw.nl/schemas/dcx/gml/"
+        xmlns:gml="http://www.opengis.net/gml"
+        xmlns:abr="http://www.den.nl/standaard/166/Archeologisch-Basisregister/"
+        xmlns:ddm={schemaNameSpace}
+        xmlns:id-type="http://easy.dans.knaw.nl/schemas/vocab/identifier-type/"
+        xsi:schemaLocation={s"$schemaNameSpace $schemaLocation"}
+      >
+        <ddm:profile>
+          { emd.getEmdTitle.getDcTitle.asScala.map(bs => <dc:title xml:lang={ lang(bs) }>{ bs.getValue }</dc:title>) }
+          { emd.getEmdDescription.getDcDescription.asScala.map(bs => <dct:description xml:lang={ lang(bs) }>{ bs.getValue }</dct:description>) }
+          { emd.getEmdDescription.getTermsAbstract.asScala.map(bs => <dct:description xml:lang={ lang(bs) } descriptionType='Abstract'>{ bs.getValue }</dct:description>) }
+          { emd.getEmdDescription.getTermsTableOfContents.asScala.map(bs => <dct:description xml:lang={ lang(bs) } descriptionType='TableOfContent'>{ bs.getValue }</dct:description>) }
+          { /* instructions for reuse not specified as such in EMD */ }
+          { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator eas:scheme={ bs.getScheme } eas:schemeId={ bs.getSchemeId } xml:lang={ lang(bs) }>{ bs.getValue }</dc:creator>) }
+          { emd.getEmdCreator.getEasCreator.asScala.map(author => <dcx-dai:creatorDetails>{ toXml(author)} </dcx-dai:creatorDetails>) }
+          { dateCreated.map(node =>  <ddm:created>{ node.text }</ddm:created>) }
+          { dateAvailable.map(node =>  <ddm:available>{ node.text }</ddm:available>) }
+          { disciplines.map(code => <ddm:audience>{ code }</ddm:audience>) }
+          <ddm:accessRights>{ emd.getEmdRights.getAccessCategory }</ddm:accessRights>
+        </ddm:profile>
+        <ddm:dcmiMetadata>
+          { emd.getEmdIdentifier.getDcIdentifier.asScala.filter(isDdmId).map(bi => <dct:identifier xsi:type={ xsiType(bi) }>{ bi.getValue }</dct:identifier>) }
+          { emd.getEmdTitle.getTermsAlternative.asScala.map(str => <dct:alternative>{ str }</dct:alternative>) }
+          { emd.getEmdRelation.getDCRelationMap.asScala.map { case (key, values) => values.asScala.map(toRelationXml(key, _)) } }
+          { emd.getEmdRelation.getRelationMap.asScala.map { case (key, values) => values.asScala.map(toRelationXml(key, _)) } }
+          { emd.getEmdContributor.getDcContributor.asScala.map(bs => <dc:contributor eas:scheme={ bs.getScheme } eas:schemeId={ bs.getSchemeId } xml:lang={ lang(bs) }>{ bs.getValue }</dc:contributor>) }
```

Comment:
as with dc:creator, i don't see what these scheme/schemeID do in a creator. Also, a language tag does not make sense to me in a name element.
```suggestion
          { emd.getEmdContributor.getDcContributor.asScala.map(bs => <dc:contributor>{ bs.getValue }</dc:contributor>) }
```